app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/BgGraphBuilder.java

```diff
@@ -1300,6 +1303,14 @@ private synchronized void addBgReadingValues(final boolean simple) {
                 }
 
             }
+
+            if (prefs.getBoolean("Libre2_showRawGraph",false)) {
```

Comment:
It would be an option to additionally check if Libre2 is the currently selected source.
I  have no strong opinion here.

Reply:
I agree, I leave it, as I think that this setting is only possible if a libre is used. The menu will be switch off if the collector is a other one. But if someone change later the colector then the setting is wrongly on (nevermind that then no data will be avalible ;) ) anyhou I will implement it

Reply:
Solved out with commit https://github.com/NightscoutFoundation/xDrip/pull/975/commits/79e519ba03cd80c81b151c3adbddca0b68569cf6